test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -400,6 +400,15 @@
          (is (= (:status response) 200))
          (is (= (:body response) hello-body))))
 
+     ;;(testing "request to jetty successful when a non-traversable directory is created in cert dir"
+     ;;   (let [parent (fs/parent tmp-file)
+     ;;         non-traversable-dir (fs/temp-dir "foo")]
+     ;;     (fs/chmod "-rwx" non-traversable-dir)
+     ;;     (fs/sym-link (fs/file parent "bar") non-traversable-dir)
+     ;;     (let [response (get-request)]
+     ;;       (is (= (:status response) 200))
+     ;;       (is (= (:body response) hello-body)))))
```

Comment:
That seems fine. The sort of test that you're describing sounds like it belongs in tk-fs-watcher anyway?